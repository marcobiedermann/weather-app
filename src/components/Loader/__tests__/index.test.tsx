import { render } from '@testing-library/react';
import React from 'react';
import { describe, expect, it } from 'vitest';
import Loader from '..';

describe('components/Loader', () => {
  it('renders correctly', () => {
    expect.assertions(1);

    const { asFragment } = render(<Loader />);

    expect(asFragment()).toMatchSnapshot();
  });
});
