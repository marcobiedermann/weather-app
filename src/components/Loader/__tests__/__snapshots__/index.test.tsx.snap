// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`components/Loader > renders correctly 1`] = `
<DocumentFragment>
  <div>
    loading â€¦
  </div>
</DocumentFragment>
`;
